packages/react-components/react-infobutton/Spec.md

```diff
@@ -1,52 +1,138 @@
 # @fluentui/react-infobutton Spec
 
+Convergence epic issue: [#25062](https://github.com/microsoft/fluentui/issues/25062)
+
 ## Background
 
-_Description and use cases of this component_
+An InfoButton provides a way for users to get more information about a particular UI element. It is a button that displays an icon and when clicked a Popover displays the additional information.
```

Comment:
Do I correctly understand that this component is basically following?

```jsx
<Popover>
  <PopoverTrigger>
    <Button icon='X' />
  </PopoverTrigger>
  <PopoverSurface>Some content</PopoverSurface>
</Popover>
```

Reply:
After an update, we are not using the v9 button, but the native button. Other than that, you are correct. It's a pretty simple component, but will bring an opinionated approach to it required for Field.

Reply:
This paragraph should be updated to make it obvious that InfoButton can contain focusable content as well (Link/button etc)